packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -59,6 +59,39 @@ def test_if_marathon_app_can_be_deployed_with_mesos_containerizer(cluster):
     cluster.marathon.deploy_test_app_and_check(app, test_uuid)
 
 
+def test_if_marathon_pods_can_be_deployed_with_mesos_containerizer(cluster):
+    """Marathon pods deployment integration test using the Mesos Containerizer
+
+    This test verifies that a Marathon pods can be deployed.
+    """
+
+    test_uuid = uuid.uuid4().hex
+
+    pod_definition = {
+        'id': '/integration-test-pods-{}'.format(test_uuid),
+        'scaling': {'kind': 'fixed', 'instances': 1},
+        'environment': {'PING': 'PONG'},
+        'containers': [
+            {
+                'name': 'ct1',
+                'resources': {'cpus': 0.1, 'mem': 32},
+                'image': {'kind': 'DOCKER', 'id': 'busybox'},
+                'exec': {'command': {'shell': 'while true; do echo the current time is $(date); sleep 1; done'}}
+            },
+            {
+                'name': 'ct2',
+                'resources': {'cpus': 0.1, 'mem': 32},
+                'exec': {'command': {'shell': 'while true; do echo -n $PING ' ';sleep 1; done'}}
+            }
+        ],
+        'networks': [{'mode': 'host'}]
+    }
+
+    instances = cluster.marathon.deploy_marathon_pod(pod_definition)
+    assert pod_definition['scaling']['instances'] == instances
```

Comment:
As written, this will leave the pod if the test fails, which could have unknown consequences for other tests. As such, I would prefer we add a context manager method like the deploy_and_cleanup one for apps.